### PR TITLE
feat(ci): enable gas traces

### PR DESCRIPTION
**Pull request by jxs**
## Description
Enables verbose and gas traces when passed the `-vv` flag, also adds support for `-v` in the cli to enable logging in the tests. 

Closes https://github.com/polyphene/kythera/issues/48 & https://github.com/polyphene/kythera/issues/52

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by tchataigner** [comment]
Closes #52

File: lib/src/lib.rs [hunk 1]
```diff
@@ -4,16 +4,17 @@
 #[cfg(feature = "colors")]
 use colored::Colorize;
 
+// Re-exports.
+// TODO: are these types really necessary to be exposed? Should we wrap them in our own?
```
Comment (PhilippeMts) [inline lib/src/lib.rs:8]: Should we remove this`TODO`comment and remind us to handle this question through a _enhancement_ ticket like [this one](https://github.com/polyphene/kythera/issues/45)?

**Review by PhilippeMts — CHANGES_REQUESTED** [review]

File: lib/src/lib.rs (hunk 1 above)
Comment (jxs) [reply lib/src/lib.rs]: yup, makes sense Philippe, opened https://github.com/polyphene/kythera/issues/59. And removed the todo. Thanks

**Review by PhilippeMts — APPROVED** [review]

**Review by tchataigner — APPROVED** [review]
LGTM !

**Pull request merged**